src/components/button/button.html

```diff
@@ -8,7 +8,13 @@
     <link rel="stylesheet" href="../../css/header.css" />
     <link rel="stylesheet" href="../../css/main.css" />
     <link rel="stylesheet" href="../../css/docs.css" />
-    <link rel="stylesheet" href="./avatar.css" />
+    <link rel="stylesheet" href="./button.css" />
+    <link
+      rel="stylesheet"
+      href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
```

Comment:
I don't know the reason but someone suggested in Review session to import the material icon link through CSS.
Syntax: @import 'file'

Reply:
Fixed, added the icons url to root.css